lib/inline-view.coffee

```diff
@@ -32,7 +32,7 @@ class InlineView
     # Config value if you want to limit the status bar report
     # if your cursor is in the range or error, or on the line
     limitOnErrorRange = (atom.config.get('linter.statusBar') ==
-                         'Show error if the cursor is in range')
+                        'Show error if the cursor is in range')
```

Comment:
@dmnd `Coffeelint` grunt task gave me an error. I suggest an other change in my new commit to fit for everyone ;)